helm/overlay/patches/patch_deployment.yaml

```diff
@@ -25,6 +26,7 @@ spec:
 #    spec:

```

Comment:
The cert-manager patch is commented out since it was an example, can you please remove the comments and make sure the image pull policy is updated for this one too? Keep the NEW_ENV commented tho

after the changes you can run 
`make helm package RELEASE_REGISTRY=yourdockerregistry `
and than go to helm/chart/charts and test the new generated chart installation with 

`helm upgrade --install -n keptn-lifecycle --create-namespace toolkit keptn-lifecycle-toolkit-0.1.0.tgz`

Reply:
updated